utils/runtime-fuzzer/src/gear_calls.rs

```diff
@@ -51,6 +52,10 @@ static_assertions::const_assert!(MAX_PAYLOAD_SIZE <= gear_core::message::MAX_PAY
 const MAX_SALT_SIZE: usize = 512;
 static_assertions::const_assert!(MAX_SALT_SIZE <= gear_core::message::MAX_PAYLOAD_SIZE);
 
+const ID_SIZE: usize = mem::size_of::<ProgramId>();
+const GAS_AND_VALUE_SIZE: usize = mem::size_of::<(u64, u128)>();
```

Comment:
I think it can be covered by the `AUXILIARY_SIZE` bytes

Reply:
I'd leave bytes for id and gas for more clarity